tests/queries/0_stateless/00900_long_parquet_load.sh

```diff
@@ -66,6 +66,6 @@ EOF
     # Some files contain unsupported data structures, exception is ok.
     cat "$DATA_DIR"/"$NAME" | ${CLICKHOUSE_CLIENT} --query="INSERT INTO parquet_load FORMAT Parquet" 2>&1 | sed 's/Exception/Ex---tion/'
 
-    ${CLICKHOUSE_CLIENT} --query="SELECT * FROM parquet_load LIMIT 100"
+    ${CLICKHOUSE_CLIENT} --query="SELECT * FROM parquet_load ORDER BY tuple(*) LIMIT 100"
```

Comment:
Why do we need it? Just curious

Reply:
The order of the rows is not deterministic anymore.

Reply:
I misread `tuple()` instead of `tuple(*)` and thought how it helps. Thanks